packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -140,6 +142,13 @@ function CustomViewShell(props: TCustomViewShellProps) {
     []
   );
 
+  useLayoutEffect(() => {
```

Comment:
This is the simplest solution.

I think I would prefer to update how we generate the HTML document when running/building the application to already include this attribute but I was worried that end-users would need to make sure they update all `app-kit` dependencies at the same time, and last year we learnt that's not always the case.

Pelase, let me know waht you think 🙏

Reply:
Thanks. That's a better approach for our current use case.

Updated here: 56946763ba1c249bf2c904492b35a4b00ffa2f22